src/startApp.js

```diff
@@ -49,15 +49,19 @@ const startApp = () => {
           )
         : Promise.resolve(),
     () => {
-      let bundle = window[appMetadata.id]
+      let bundle = window[appMetadata.safeId]
       // support rollup and esbuild
       if (typeof bundle !== 'function') {
         bundle = bundle.default
       }
 
       console.time('app2')
-      settings.appSettings.version = appMetadata.version
+      //Adding complete metadata info to app settings
+      settings.appSettings = { ...appMetadata }
```

Comment:
I think if we use a simple `Object.assign` it will yield the same effect, without generating more code in the compiled (es5) version. Can you check this?